micrometer-core/src/test/java/io/micrometer/core/instrument/binder/jetty/InstrumentedQueuedThreadPoolTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019 VMware, Inc.
```

Comment:
Should be 2024.
Could you please also update the year in other files too?

Reply:
Yes. is `VMware, Inc.` still the correct copyright holder?

Reply:
Yes, thanks.